common/src/main/java/com/microsoft/identity/common/internal/cache/IAccountCredentialCache.java

```diff
@@ -109,7 +110,8 @@
             final String clientId,
             final String realm,
             final String target,
-            final String authScheme
+            final String authScheme,
+            final String requestedClaims
```

Comment:
Hmmm we'll likely want to make this an additive API change... we have a few different consumers of common, so it might make sense to introduce a new overload that is called with `null` as a param by the previous (older) version of the method

Reply:
Please take a look at what I did in the latest commit, I think this should address your concerns :)